storage/dynamodbStorage.js

```diff
@@ -47,6 +47,10 @@ module.exports = class DynamodbStorage {
   }
 
   loadDependencies (dependencies) {
+    if (dependencies.length === 0) {
+      return Promise.resolve(dependencies)
```

Comment:
Should probably just `Promise.resolve([])` in this case, even though it will functionally do the same thing, since this only happens if `dependencies` is empty. The reason being that `insertTasks` always returns the tasks its given in the argument (for some reason, I don't really know why it was done like that), so resolving to `tasks` makes sense. Whereas this function returns something different to its argument, so writing `Promise.resolve(dependencies)` is a little misleading.